Pull request try #2

kellyschroeder opened the pull request:
Hopefully this works this time... I cannot seem to find where I am going wrong.

kellyschroeder committed:
Added name, email, and phone number

kellyschroeder committed:
try #2

kellyschroeder committed:
Update README.md

mdietze commented:
Ok, so Andre's suggestion here is WRONG -- you shouldn't ever have to fork again and doing so will overwrite your own repository. What you need to do is pull from the main repository: _git pull https://github.com/EcoForecast/TeamGOATS master_. Indeed, Kelly didn't even need to close her earlier pull request. Other people have changed the same file and those changes will need to be resolved with the changes she's made. From git's perspective it's seeing multiple edits to the same lines in a file and it can't figure out which version of the edits is correct, so it's telling you that it needs to be sorted out manually. Sometimes file conflicts are unavoidable, so learning how to resolve them is an important lesson! That said, at other times file conflicts can be avoided by merging pull requests promptly.

**GENERAL LESSON TO EVERYONE**: alway do a git pull from the OWNER before pushing to github and submitting a pull request to insure that your code is up-to-date with the latest code.

kellyschroeder committed:
Commit changes

kellyschroeder committed:
Merge branch 'master' of github.com:kellyschroeder/TeamGOATS

kellyschroeder committed:
commit

kellyschroeder committed:
finally

Merge branch 'master' of https://github.com/EcoForecast/TeamGOATS

# Conflicts:
#	README.md
